frontend/src/main/scala/akka/viz/frontend/FrontendApp.scala

```diff
@@ -123,38 +126,62 @@ object FrontendApp extends JSApp with Persistence
   val messagesPanel = new MessagesPanel(selectedActors)
   val userIsEnabled = Var(false)
   val onOffPanel = new OnOffPanel(_eventsEnabled, userIsEnabled)
+  val connectionAlert = new Alert()
 
   @JSExport("toggleActor")
   def toggleActor(name: String) = actorSelector.toggleActor(name)
 
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Method is longer than 50 lines](https://www.codacy.com/app/lustefaniak/akka-viz/file/2050919478/issues/source?bid=3121282&fileBranchId=3133751#l134)